components/camera/server_test.go

```diff
@@ -176,9 +176,8 @@ func TestServer(t *testing.T) {
 		imageReleasedMu.Lock()
 		test.That(t, imageReleased, test.ShouldBeTrue)
 		imageReleasedMu.Unlock()
-		test.That(t, resp.MimeType, test.ShouldEqual, utils.MimeTypeRawRGBA)
+		test.That(t, resp.MimeType, test.ShouldEqual, utils.MimeTypeJPEG)
```

Comment:
Create another test where the mime type requested is explicitly MimeTypeRawRGBA, and see if you get back the expected raw RGBA image

Reply:
Someone (probably you) already wrote that exact test above this one. I just threw some comments in there to maybe make it more obvious